src/main/java/com/passpnu/passwordmanager/entity/Password.java

```diff
@@ -0,0 +1,26 @@
+package com.passpnu.passwordmanager.entity;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class Password {
+    @Id
+    @GeneratedValue(strategy= GenerationType.AUTO)
+    private Long id;
+    @NotBlank
+    @ManyToOne
```

Comment:
I'm not sure that just specifying `@ManyToOne` will work. Did you test it?

Reply:
Specified table columns which links the entities